srv/components/csm/render_ha_template.sls

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
@83bhp 
Why have we created this new file?

Reply:
To be able to run "Render CSM ha input params template" state explicitly during unboxing.
If csm.prepare is called the repos get created.